descriptors/ModuleDescriptor-template.json

```diff
@@ -4,7 +4,7 @@
   "provides": [
     {
       "id": "inventory",
-      "version": "9.1",
+      "version": "9.3",
```

Comment:
Why has this been incremented by two minor versions?

Reply:
PRs related assigning tags to instance records changed version of inventory interface from 9.1 to 9.2. And PRs about assigning tags to item records will update version to 9.3.